egs/fisher_swbd/s5/path.sh

```diff
@@ -1,4 +1,4 @@
-export KALDI_ROOT=`pwd`/../../..
+export KALDI_ROOT=../../kaldi
```

Comment:
Don't change the standard, which is the same in all recipes. Also fix this in multi-en and elsewhere you changed.